Doc/library/itertools.rst

```diff
@@ -464,6 +465,23 @@ loops that truncate the stream.
                       pass
 
 
+.. function:: ilen(iterable)
+
+   Consume the iterator and return the number of consumed elements.
```

Comment:
You are talking about an *iterator* but your argument is an *iterable*. I would phrase it like this:

```
Return the length of an *iterable*.

If *iterable* is an iterator, it is consumed.
```